src/routes/safe/components/Apps/components/AddAppForm/index.tsx

```diff
@@ -33,8 +33,11 @@ const AppInfo = styled.div`
 const AppDocsInfo = styled.div`
   display: flex;
   margin-bottom: 10px;
-  a {
-    margin-left: 5px;
+  flex-direction: column;
+  svg {
+    position: relative;
+    top: 5px;
```

Comment:
Could we fix this values in order to center and give more space to the left? 
top: 4px;
left: 4px;